cms/templates/js/validation-error-modal.underscore

```diff
@@ -1,18 +1,19 @@
 <div class = "validation-error-modal-content">
     <div class "error-header">
         <p>
-        <%= _.template(
-        ngettext(
-            "There was {strong_start}{num_errors} validation error{strong_end} while trying to save the course settings in the database.",
-            "There were {strong_start}{num_errors} validation errors{strong_end} while trying to save the course settings in the database.",
-            num_errors
-        ),
-        {
-            strong_start:'<strong>',
-            num_errors: num_errors,
-            strong_end: '</strong>'
-        },
-        {interpolate: /\{(.+?)\}/g})%>
+        <%= interpolate(
```

Comment:
Note to reviewers: Here I got rid of _.template in favor of interpolate. I could revert this change though now that I understand how to pass settings to _.template with the new syntax.

However, I did manually test that the new code works, and it seems like a simplification to me, so I vote for keeping it.